client/dashboard/app/root/index.tsx

```diff
@@ -24,18 +25,18 @@ const VERY_SLOW_THRESHOLD_MS = 6000;
 function Root() {
 	const { name, LoadingLogo = WordPressLogo } = useAppContext();
 	const isFetching = useIsFetching();
-	const router = useRouterState();
-	const routeMeta = useRouterState( {
-		select: ( state ) => {
-			return state.matches.map( ( match ) => match.meta! ).filter( Boolean );
-		},
-	} );
+	const router = useRouter();
```

Comment:
`useRouter` is non-reactive. We just want a non-changing reference to the router so we can refer to it when the timer expires.

To avoid the confusion between this line and the existing `const router = useRouterState()` I merged all the existing reactive access of router state into the one `select` function.